src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -107,7 +105,7 @@ class PolicyHandler
 
   std::string GetLockScreenIconUrl() const;
   void ResetRetrySequence();
-  int NextRetryTimeout();
+  uint32_t NextRetryTimeout();
```

Comment:
If you use `std::uint32_t` type - you should include `cstdint` to make the version in the namespace std available (and possibly the one in the global namespace). Including `stdint.h` should just make the unqualified version available - `uint32_t`.
They are the same and serve the same purpose. If you want to use the version in the std namespace, include `cstdint`. If you want the one in the global namespace, include `stdint.h`. For C++ it is recommended to use the one in the `std` namespace. As a rule: Always include what you use and don't rely on other headers including things for you.